packages/text-input/tsconfig.json

```diff
@@ -19,11 +19,17 @@
     {
       "path": "../icon"
     },
+    {
+      "path": "../interaction-ring"
+    },
     {
```

Comment:
Can we remove this now, since we're not explicitly relying on the package anymore?

Reply:
👍